Trappist/src/Promact.Trappist.Web/wwwroot/app/questions/questions-dashboard/delete-question-dialog.html

```diff
@@ -0,0 +1,13 @@
+﻿<div class="dialog">
+    <div class="dialog-title clearfix">
```

Comment:
As discussed with you, using their directives does not suffice our design needs and I have to override their styles so I skipped using their directives. But anyways now as you say, I will use their directives and customize styles as per our need.